stl/inc/exception

```diff
@@ -31,7 +31,15 @@ _STD_END
 #if _HAS_EXCEPTIONS
 
 #include <malloc.h>
+
+// TRANSITION, terminate_function and unexpected_function are defined for source compatibility.
+#pragma push_macro("terminate_function")
+#pragma push_macro("unexpected_function")
+#undef terminate_function
+#undef unexpected_function
 #include <vcruntime_exception.h>
+#pragma pop_macro("unexpected_function")
+#pragma pop_macro("terminate_function")

```

Comment:
This seems fragile against other inclusions of public VCRuntime headers. I suggest you remove all changes regarding `terminate_function` and `unexpected_function`, I've opened MSVC-PR-519139 to change `eh.h` directly.